src/app/util/af-types.h

```diff
@@ -235,6 +235,15 @@ struct EmberAfDefinedEndpoint
      * Span pointing to a list of tags. Lifetime has to outlive usage, and data is owned by callers.
      */
     chip::Span<const chip::app::Clusters::Descriptor::Structs::SemanticTagStruct::Type> tagList;
+
+#ifdef CONFIG_USE_ENDPOINT_UNIQUE_ID
```

Comment:
Here we absolutely have to make sure this header includes the right config header.  Otherwise we can have different translation units disagree on the layout of this struct!